src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -433,7 +539,11 @@
     /** The function that compares paths converging on the same vertex to decide which ones continue to be explored. */
     public DominanceFunction dominanceFunction = new DominanceFunction.Pareto();
 
-    /** Accept only paths that use transit (no street-only paths). */
+    /** Accept only paths that use transit (no street-only paths).
+     *
+     * @Deprecated - This will probably be reimplemented
+     * */
+    @Deprecated
     public boolean onlyTransitTrips = false;
```

Comment:
I am not sure what this is? Is this just a special case of combining different modes?

Reply:
It is used as part of the a star algorithm. To get a variety if itineraries, you ban the trips that were found previously and then to another search. If the first trip you find is walk-only, then you need to ban walk-only in order to not find the same itinerary again. In Raptor this setting is implied and a walk-only search needs to be done separately.